source/payment_flow/index.html.md.erb

```diff
@@ -58,9 +58,11 @@ When your service redirects your user to `next_url`, they’ll see an **Enter ca
 * billing address
 * email address
 
+If your service takes [digital wallet payments](/digital_wallets/), your user can also choose to pay with Apple Pay or Google Pay from this page.
```

Comment:
The more I read it, I think you're right in saying 'from this page', but when I first read it for some reason I stumbled over it. It's not supposed to say 'on the same page' or 'on this page', right?

Reply:
Oh yeah, now that you mention it, it does feel a little bit weird.

I think I went with 'from' because if a user selects Google Pay or Apple Pay, it opens up a separate window to complete the payment. I think 'from' is okay but I'll run it past our content designer as well.

Thank you!